scr/main/java/bittercode/model/Cart.java

```diff
@@ -0,0 +1,31 @@
+package com.bittercode.model;
+
+import java.io.Serializable;
+
+public class Cart implements Serializable {
+
+    private Book book;
+    private int quantity;
+
+    private Cart(Book book, quantity) {
```

Comment:
<div>


<div id="suggestion">
<div id="issue"><b>Missing type declaration for constructor parameter</b></div>
<div id="fix">

The constructor parameter `quantity` is missing its type declaration. Consider adding `int` before `quantity`.
</div>
<details>
<summary>
<b>Code suggestion</b>
</summary>
<div id="code">

```
 @@ -10,1 +10,1 @@
 -    private Cart(Book book, quantity) {
 +    private Cart(Book book, int quantity) {
```
</div>
</details>
</div>


<small><i>Code Review Run <a href=https://github.com/PrajaktaBendreBito/BookStore/pull/3#issuecomment-2434572244>#7fa0a4</a></i></small>
</div>


---
🚩 Problem result:
- [ ] This issue was flagged incorrectly